.expeditor/update_chef-workstation-app_to_latest.sh

```diff
@@ -13,11 +13,10 @@
 set -evx
 
 version=$(get_github_file $REPO master VERSION)
-
 branch="expeditor/chef_workstation_app_${version}"
 git checkout -b "$branch"
 
-sed -i -r "s/override :\"chef-workstation-app\",\s+version: \"v[^\"]+\"/override :\"chef-workstation-app\", version: \"v${version}\"/" omnibus/config/projects/chef-workstation.rb
+sed -i -r "s/^default_version .*/default_version \"v${version}\"/" omnibus/config/projects/chef-workstation.rb
```

Comment:
I think the path here needs to be `omnibus/config/software/chef-workstation-app.rb`